### ISSUE TITLE
 owner_id in oauth_client 

### ISSUE THREAD

**Issue by kinastowski** [tsuyoshizawa/scala-oauth2-provider-example-skinny-orm#1]
Hi, I'm studing your code right now and I was wondering what's the point of owner_id in oauth_client class, as it is not verified anywhere. Do oauth_client realy needs an owner?
thanks, /<.

**Comment by hubino-testing2** [comment]
Yes. The above comment is correct according to me too. Because, oauth_client do not need owner anywhere.
Also, Each users should be mapped under one client. This mapping is not handled in the example.
